apps/busybox/deployers/busybox_statefulset.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    lkey: lvalue
+  name: busybox
+spec:
+  clusterIP: None
+  selector:
+    lkey: lvalue
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: busybox
+  labels:
+    lkey: lvalue
+spec:
+  serviceName: busybox
+  rkey: rvalue
+  selector:
+    matchLabels:
+      lkey: lvalue
+      #The Label "openebs.io/replica-anti-affinity" 
+      #needs to be added in order to distribute volume replicas.
+      openebs.io/replica-anti-affinity: lvalue
+  template:
+    metadata:
+      labels:
+        lkey: lvalue
+        openebs.io/replica-anti-affinity: lvalue
+    spec:
+      containers:
+      - name: busybox
+        image: ubuntu
```

Comment:
This image should be busybox or ubuntu?

Reply:
@dargasudarshan changed to  busybox image